apps/meteor/client/sidebar/sections/components/OmnichannelMakeCall.tsx

```diff
@@ -0,0 +1,31 @@
+import { Sidebar } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { ReactElement } from 'react';
+
+import { useOutboundDialer } from '../../../../ee/client/hooks/useOutboundDialer';
+
+export const OmnichannelMakeCall = (): ReactElement => {
+	const voipData = useOutboundDialer();
+
+	const getTooltip = (): string => 'Click to dial 70010';
+
+	const getIcon = (): 'phone' => 'phone';
+
+	const getColor = (): 'success' => 'success';
+
+	const voipCallIcon = {
+		title: getTooltip(),
+		color: getColor(),
+		icon: getIcon(),
+	} as const;
+
+	// TODO: move registration flow to context provider
+	const makeCall = useMutableCallback((): void => {
```

Comment:
Will we merge this PR without the `TODO` done? Or is this another "test" file? If so pls comment it

Reply:
@MartinSchoeler This file is a for demonstrating how to use makeCall function. Once the dialpad is in place, this file should be removed. I will add such comment to that file.